### PR TITLE
Add 1Panel icon

### LINKED ISSUE #10396: Request: 1Panel
### Brand Name

1Panel

### Website

https://1panel.cn

### Popularity Metric

[Similarweb global rank](https://www.similarweb.com/website/1panel.cn) for 1panel.cn: 289,963

The number of [GitHub stars](https://github.com/1Panel-dev/1Panel) : 15.2k

### Official Resources for Icon and Color

Logo with title (PNG) : http://1panel.oss-cn-hangzhou.aliyuncs.com/img/1panel-logo.png (from GitHub README.md)

High-Resolution logo with title (PNG) : https://1panel.cn/img/logo-blue.png (from website title)

High-Resolution icon-only logo (PNG) : https://1panel.cn/img/favicon.png (from website favicon)

Icon-only logo (White, PNG) : https://1panel.cn/docs/img/logo-white.png (from docs title)

Color: #005eeb

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by yhmyhm0910**
**Issue:** closes #10396

**Similarweb rank: 289,963** 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

  - Hex value picked using online color picker with the most area covered.
  - PNG to Final SVG: Trace bitmap in Inkscape -> Optimize with a precision of 3 in SVGOMG.
  - No multiple sources used.

### CODE REVIEW

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Thanks for the contribution, @yhmyhm0910 👍🏻 Frustrating that there's no vector version available; I was able to find ones for pretty much all of their parent company's products, but not this one.

The first thing that jumps out to me is that I don't think we need to include the gaps; the colour differences in the source are subtle enough that it's still identifiable when simply flattened to a single colour.

The next thing to note is that the source isn't a perfect square; it's taller than it is wide, so your SVG should be 24dp high with the width scaled relative to that and then centred on the canvas.

Finally, what have you use to vectorise the PNG? Running it through [Convertio](https://convertio.co) gives a pretty good starting point, but it will still need to some manual work to get it to line up perfectly and remove some excess points.

**Comment by yhmyhm0910** [comment]
> The first thing that jumps out to me is that I don't think we need to include the gaps; the colour differences in the source are subtle enough that it's still identifiable when simply flattened to a single colour.

I'll try to delete it.

> The next thing to note is that the source isn't a perfect square; it's taller than it is wide, so your SVG should be 24dp high with the width scaled relative to that and then centred on the canvas.

Does that mean the viewBox of the SVG need not to be "0 0 24 24"?

> Finally, what have you use to vectorise the PNG? Running it through [Convertio](https://convertio.co) gives a pretty good starting point, but it will still need to some manual work to get it to line up perfectly and remove some excess points.

I use Inkscape to trace bitmap. Didn't change any default parameters.

**Comment by adamrusted** [comment]
Are you still working on this @yhmyhm0910?

File: icons/1panel.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...25s-.029.011-.029.025c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.117-.05c0-.015-.013-.026-.03-.026-.016 0-.029.011-.029.025 0 .014.013.026.03.026.016 0 .029-.012.029-.026zm-.175-.102c0-.014-.014-.025-.03-.025s-.029.011-.029.025c0 .014.013.025.03.025.015 0 .029-.011.029-.025zm-.117-.05c0-.014-.013-.026-.03-.026-.016 0-.029.012-.029.025 0 .014.013.026.03.026.016 0 .029-.012.029-.026zm-.117-.051c0-.014-.013-.025-.03-.025-.015 0-.028.01-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm-.175-.101c0-.014-.013-.026-.03-.026-.016 0-.029.012-.029.026 0 .013.013.025.03.025.016 0 .029-.012.029-.025zM2.1 7.024c0-.014-.013-.025-.03-.025-.015 0-.028.011-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm-.117-.05c0-.015-.013-.026-.029-.026s-.03.011-.03.025c0 .014.014.026.03.026s.03-.012.03-.026zm-.175-.102c0-.014-.013-.025-.03-.025-.015 0-.028.011-.028.025 0 .014.013.025.029.025s.029-.011.029-.025zm9.315 10.226-1.227-.616-.015-3.4-.015-3.4H8.521l.016-.67.016-.67 1.898-.948 1.898-.948.015 2.838c.008 1.56.008 4.096 0 5.633l-.015 2.796zm1.314-5.04V6.433l.277.14c.153.077.324.164.38.194.096.051.102.36.101 5.322v5.268l-.346.165c-.19.09-.36.164-.379.164-.018 0-.033-2.532-.033-5.626z"/></svg>
```
Comment (adamrusted) [inline icons/1panel.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>1Panel</title><path d="m12 0 10.349 6v12L12 24 1.651 18V6zm0 .326L1.897 6.158v11.664L12 23.653l10.103-5.831V6.158zM8.84 20.523l-5.801-3.349V6.826L12 1.653l2.23 1.287-8.925 5.195v7.73l5.792 3.345zm6.299-17.058 5.822 3.361v10.348L12 22.347l-2.274-1.312 8.969-5.17v-7.73l-5.823-3.362zm-2.137 3.35v2.869l.024 7.666-.691.384-2.18-1.249.008-6.801H8.958L8.95 8.351l3.412-1.965z"/></svg>
```

I've had a play with manually vectorising this based on the PNG in the header [here](https://1panel.cn/docs/). Could @uncenter or another member of the team review, and if happy - can we merge it in?

Comment (uncenter) [reply icons/1panel.svg:1]: Looks quite good, go ahead and commit it and I'll approve :)

Comment (uncenter) [reply icons/1panel.svg:1]: For the record, here is how it compares (first with the PNG logo behind the icon, second with the PNG logo in front of the icon):
Screenshot 2024-06-04 at 15 51 54 (Inkscape)
Screenshot 2024-06-04 at 15 51 48 (Inkscape)

**Review by uncenter — APPROVED** [review]

**Pull request merged**